src/ogt_voxel_meshify.h

```diff
@@ -688,6 +697,12 @@ ogt_mesh* ogt_mesh_from_paletted_voxels_simple(
                     mesh->vertex_count += 4;
                     mesh->index_count  += 6;						
                 }
+
+                uint32_t count = mesh->vertex_count - start_vert_count;
```

Comment:
I'm unsure that the context is the right place for the callback because it might seem to users to be useful for more than just the ogt_mesh_from_paletted_voxels_simple function. 

Instead, I would favor creating a second version of the _simple function that receives the callback and context as a parameter, and then point the one without the parameter to this one like so:

```
ogt_mesh* ogt_mesh_from_paletted_voxels_simple( <current-args>) {
    return ogt_mesh_from_paletted_voxels_simple( <current-args>, NULL, NULL );
}

ogt_mesh* ogt_mesh_from_paletted_voxels_simple( <current-args>, ogt_voxel_meshify_simple_vertex_func func, void* func_context) {
   ... the current code for ogt_mesh_from_paletted_voxels_simple as usual
}
```

Reply:
I definitely see your point. With my latest changes though It adds another function pointer as a parameter. I'm worried that maybe the argument list would be long to the point of being cumbersome. What do you think?